ridgeback_control/config/control.yaml

```diff
@@ -40,3 +40,21 @@ ridgeback_velocity_controller:
       max_velocity           : 2.0   # rad/s
       has_acceleration_limits: true
       max_acceleration       : 1.0   # rad/s^2
+
+ekf_localization:
+  frequency: 50
+  odom0: /ridgeback_velocity_controller/odom
+  odom0_config: [false, false, false,
+                 false, false, false,
+                 true, true, true,
+                 false, false, true]
+  odom0_differential: true
```

Comment:
Sorry, I received an e-mail notification for this, so I thought I'd check it out.

Technically, the 12-D configuration specification is deprecated. It might be good to add the last three (acceleration) parameters, even if they're false. Also, you're not fusing any of the pose variables, so turning on differential mode won't have any effect here.